### PR TITLE
Fix Auto Play - Clicked Dots Remains active #1432

### LINKED ISSUE #1432: Auto Play - Clicked Dots Remains active
In the Auto Play demo page (https://react-slick.neostack.com/docs/example/auto-play-methods).
When we click on a dot, the user is getting a feeling like two dots are active. 
Why are we having this behaviour ?

image

### PR DESCRIPTION
**Pull request by Caps-Looking**
Fix #1432 
As react-slick uses slick-carousel's css, changing the css doc's file wouldn't fix the error for library users, it would only fix for the docs.
I created a css file and a webpack loader to use it. This fix every use of the component.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ module.exports = {
       {
         test: /\.md$/,
         loader: "html!markdown"
+      },
+      {
+        test: /\.css$/,
+        use: "css-loader"
```
Comment (hjJunior) [inline webpack.config.js:29]: I guess you have missed adding the package.json

File: src/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.slick-dots li.slick-active button:focus:before,
+.slick-dots li button:hover:before
+{
+    opacity: 1;
+}
+
+.slick-dots li button:focus:before
+{
+    opacity: 0.25;
+}
```
Comment (hjJunior) [inline src/css/styles.css:10]: Pls add a new empty line on the end of files

File: webpack.config.js (hunk 1 above)
Comment (Caps-Looking) [reply webpack.config.js:29]: Nope, the css-loader is already on the package.json

**Comment by akiran** [comment]
We can't use css file fix for this issue.

We will find a different solution for this problem

**Pull request closed without merging**